mod/calculated_metrics/lib/formula/calculator/invalid_input.rb

```diff
@@ -0,0 +1,14 @@
+module Formula
+  class Calculator
+    # Placeholder for the case that formula contains input items that don't exist
```

Comment:
I like this.  Probably good to have it before we recruit a lot more people to write formulae!